BankChain/app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,8 @@
     <!-- Permission to talk to banks -->
     <uses-permission android:name="android.permission.INTERNET" />
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+    <uses-permission android:name="android.permission.DISABLE_KEYGUARD"/>
```

Comment:
Are these permissions needed? We don't run on Travis.
Might be useful to add an XML comment about these two, for future reference.